### PR TITLE
remove 'co-*' classes supporting PF5 imports/variables

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://bugzilla.redhat.com/show_bug.cgi?id=2256085

Screenshot 2023-12-28 at 6 22 03 PM

Screenshot 2023-12-28 at 6 22 19 PM

Screenshot 2023-12-28 at 6 22 49 PM

Screenshot 2023-12-28 at 6 22 58 PM

Screenshot 2023-12-28 at 6 23 13 PM

### CODE REVIEW

File: packages/client/src/components/dashboards/cards/activity-card.tsx [hunk 1]
```diff
@@ -115,14 +116,16 @@ export const ActivityCard: React.FC = React.memo(() => {
...
+    <Card className="odf-overview-card--gradient">
       <CardHeader>
         <CardTitle>{t('Activity')}</CardTitle>
       </CardHeader>
-      <ActivityBody className="odf-client-activity-card__body">
-        <OngoingActivity />
-        <RecentEvent />
-      </ActivityBody>
+      <CardBody>
+        <ActivityBody className="odf-client-activity-card__body">
```
Comment (GowthamShanmugam) [inline packages/client/src/components/dashboards/cards/activity-card.tsx:124]: if the ActivityBody is affected means, by any chance OCP SDK will fix this?

Comment (SanjalKatiyar) [reply packages/client/src/components/dashboards/cards/activity-card.tsx:124]: not sure, but I think that might not be needed... using `CardBody` is anyway something good to do...

**Review by bipuladh — APPROVED** [review]

**Pull request merged**